setup.md

```diff
@@ -24,78 +24,144 @@ title: Setup
+[anaconda-windows]: https://www.anaconda.com/download/#windows
+[jupyter]: http://jupyter.org/
+[python]: https://python.org
+[video-mac]: https://www.youtube.com/watch?v=TcSAln46u9U
+[video-windows]: https://www.youtube.com/watch?v=xxQ0mzZ8UvA
+
+## Opening a Conda-enabled Terminal and Verifying the Installation:
+
+### Windows
+Click Start, search, or select Anaconda Prompt from the menu. A window should pop up where you can now type commands such as checking your Conda installation with:
 
 ~~~
-conda install -c conda-forge plotnine
+conda --help
 ~~~
 {: .language-bash}
 
-### Miniconda installation
+### Mac OSX
 
-Miniconda is a "light" version of Anaconda. If you install and use Miniconda
-you will also need to install the workshop packages.
+Click the Launchpad icon in the Dock, type Terminal in the search field, then click Terminal. 
+A window should pop up where you can now type commands such as checking your conda installation with:
 
-#### Download and install Miniconda
+~~~
+conda --help
+~~~
+{: .language-bash}
 
-Download and install [Miniconda](https://docs.conda.io/en/latest/miniconda.html)
-following the instructions. Remember to download and run the installer for
-Python 3.x.
+### Linux
+This depends a bit on your Linux distribution, but often you will have an Applications listing in which you can select a Terminal icon you can click. A window should pop up where you can now type commands such as checking your conda installation with:
 
-#### Check the installation of Miniconda
+~~~
+conda --help
+~~~
+{: .language-bash}
+
+
+## Required Python Packages for this Workshop
+The following are packages needed for this workshop:
+
+* [Pandas](http://pandas.pydata.org/)
+* [Jupyter notebook](http://jupyter.org/)
+* [Numpy](http://www.numpy.org/)
+* [Matplotlib](http://matplotlib.org/)
+* [Plotnine](https://plotnine.readthedocs.io/en/stable/)
 
-From the terminal, type:
+All packages apart from plotnine will have automatically been installed with Anaconda and we can use anaconda as a package manager to install the missing `plotnine` package:
+You need to open up a *Terminal*, if you are using Mac OSX, or Linux (see instructions above), or launch an *anaconda-promt*, if you are using Windows. In your terminal window type the following: 
 
 ~~~
-conda list
+conda install -y -c conda-forge plotnine
 ~~~
 {: .language-bash}
 
-### Install the required workshop packages with conda
+This will then install the latest version of plotnine into your conda environment. 
 
-From the terminal, type:
+## Installing other Packages (Not required)
+If you want to install any additional packages, you can do so by opening a terminal window and type:
+~~~
+conda install -y package_name
+~~~
+{: .language-bash}
 
+You may need to install the required packages in this way, if you opted for installing Miniconda, instead of Anaconda. Miniconda is a "light" version of Anaconda. If you install and use Miniconda
```

Comment:
```suggestion
You may need to install the required packages in this way,
if you opted for installing Miniconda, instead of Anaconda.
Miniconda is a lightweight version of Anaconda. If you install and use Miniconda
```